pyinfra/api/config.py

```diff
@@ -1,8 +1,11 @@
+import importlib.metadata
```

Comment:
```suggestion
try:
    from importlib_metadata import entry_points
except ImportError:
    from importlib.metadata import entry_points
```

Reply:
Whoops, missed that since this import was a late addition. Thanks!

Reply:
Oh wait, I also didn't notice at first that this changed from an `import ...` to a `from ... import ...`. Does this project's coding convention dictate that this should be the latter form?

Reply:
For now I pushed a change that just does an `import` (because the module accesses a few different names from `importlib_metadata`) but I'll change it if necessary.

Reply:
Oops my bad, sorry about that!